packages/blocks/src/lib/navigation-map/index.svelte

```diff
@@ -55,6 +59,7 @@ $: $tabStore = tab;
 $: $tabsStore = tabs;
 $: $waypointsStore = waypoints;
 $: $obstaclesStore = obstacles;
+$: $plansStore = plans;
```

Comment:
Not new to this PR, but this sort of looks like this component should be providing a context rather than using singleton stores. It looks like multiple `NavigationMap` components might not be able to co-exist

Reply:
Yep, there's a bug ticket for this, with solution being move to context :)